src/components/CentralizedExchanges/index.tsx

```diff
@@ -97,32 +92,32 @@ const EthExchanges = () => {
         fontWeight={600}
         lineHeight={1.4}
       >
-        <Translation id="page-get-eth-exchanges-header" />
+        {t("page-get-eth-exchanges-header")}
       </OldHeading>
       <Text maxW="container.sm" mb={8} lineHeight={1.4} textAlign="center">
-        <Translation id="page-get-eth-exchanges-intro" />
+        {t("page-get-eth-exchanges-intro")}
       </Text>
-      <StyledSelect
+      <Select
         aria-label={t("page-get-eth-exchanges-header")}
         className="react-select-container"
         classNamePrefix="react-select"
-        options={exchangesByCountry}
-        onChange={handleSelectChange}
+        options={selectOptions}
+        onChange={handleSelectChange as any} // TODO: Fix typing
```

Comment:
@pettinarip Ran into type errors here... the prop type for `handleSelectChange` was altered slightly and now it's complaining about the type not matching for `onChange`... it's working as is, but would prefer to not have the use of `any` here